build.sbt

```diff
@@ -60,8 +60,8 @@ TwirlKeys.templateImports ++= Seq(
 routesImport += "model.editions._"
 
 val awsVersion = "1.11.999"
-val capiModelsVersion = "15.6"
-val capiClientVersion = "17.1"
+val capiModelsVersion = "17.1.0"
+val capiClientVersion = "17.21"
```

Comment:
Are these changes intended to be in this PR? They're of a different concern to me and I'd suggest they belong in their own PR.

Reply:
No quite right these are unrelated, thanks will revert them 👍